### PR TITLE
Target .NET Standard 2.0 and add net461 testing

### PR DESCRIPTION
**Pull request by natemcmaster**
cc @JunTaoLuo @Eilon

### CODE REVIEW

File: samples/LargeResponseApp/LargeResponseApp.csproj [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\Microsoft.AspNetCore.Server.Kestrel\Microsoft.AspNetCore.Server.Kestrel.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
```
Comment (JunTaoLuo) [inline samples/LargeResponseApp/LargeResponseApp.csproj]: I thought we started to use Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'" instead

Comment (natemcmaster) [inline samples/LargeResponseApp/LargeResponseApp.csproj]: We did in common.props but not samples

Comment (JunTaoLuo) [inline samples/LargeResponseApp/LargeResponseApp.csproj]: who not?

Comment (natemcmaster) [inline samples/LargeResponseApp/LargeResponseApp.csproj]: idk

**Comment by natemcmaster** [comment]
It seems something has broken SSL for .NET Framework. Will investigate more Monday.

**Pull request merged**